### DISCUSSION TITLE
Zookeeper pod is failing

### DISCUSSION THREAD

**Discussion by KarthigaSivaraman** [strimzi/strimzi-kafka-operator#9988] [Q&A]
In our cluster , strimzi is used to setup kafka with 3 replicas of zookeeper.
Out of 3 , one of them "cdc-zookeeper-2" was restarting often with issue as "Ncat : connection refused".

As we unable to predict the issue & due to lack of time . We dismantled and created new cluster . This time cdc-zookeeper-2 started working fine after 264 times restart.

What would be the root cause ?

**Comment by scholzj** [comment]
Some networking issue?
